assets/js/views/ChargingSessions.vue

```diff
@@ -220,16 +223,14 @@ export default {
 			date.setFullYear(year);
 			return this.fmtMonthYear(date);
 		},
-		showDetails(session) {
-			this.selectedSession = session;
+		showDetails(sessionId) {
+			this.selectedSessionId = sessionId;
 			const modal = Modal.getOrCreateInstance(document.getElementById("sessionDetailsModal"));
 			modal.show();
 		},
-		async sessionChanged(session) {
+		async sessionChanged(sessionId) {
 			await this.loadSessions();
-			if (session) {
-				this.selectedSession = this.sessions.find((s) => s.id == session.id);
-			}
+			this.selectedSessionId = sessionId;
```

Comment:
Ich glaub, dass du die `sessionId` hier sogar weglassen kannst. `this.selectedSessionId` ändert sich ja in der Zwischenzeit nicht.

Reply:
Stimmt. Ich dachte ich brauche die Zuweisung für den Fall des Löschen des Fahrzeugs